packages/atlas/src/views/viewer/VideoView/MoreVideos.tsx

```diff
@@ -17,17 +17,25 @@ type MoreVideosProps = {
   channelName?: string | null
   categoryId?: string
   categoryName?: string | null
+  videoId?: string
   type: 'channel' | 'category'
 }
 
-const NUMBER_OF_VIDEOS = 6
+const NUMBER_OF_VIDEOS = 7
```

Comment:
I suggest we keep it as 6 and use for both placeholder and slicing (you currently have 6 explicit there) and just add one for fetched items with maybe small comment to describe why we get additional one